run

```diff
@@ -17,12 +17,15 @@
 init_globals() {
     if [ "$0" != "/bin/bash" ]; then
         SCRIPT_DIR=$(dirname "$(readlink -f "$0")")
```

Comment:
Should we reuse `RUN_SCRIPT_FILE` to inform `SCRIPT_DIR`?

```suggestion
        RUN_SCRIPT_FILE="$(readlink -f "$0")"
        SCRIPT_DIR="$(dirname "'"'"${RUN_SCRIPT_FILE}"'"'")"
```

Reply:
I'll reuse it, but I think we don't need as many nested quotation marks -- I'll push up a fix